src/core/replication_fns/rule_fns.rs

```diff
@@ -192,3 +228,17 @@ pub fn in_place_as_deserialize<C: Component>(
     *component = (deserialize)(ctx, cursor)?;
     Ok(())
 }
+
+/// Default component consume function.
+///
+/// This implementation just calls deserialization function and ignores its result.
+pub fn consume_as_deserialize<C: Component>(
+    deserialize: DeserializeFn<C>,
+    ctx: &mut WriteCtx,
+    cursor: &mut Cursor<&[u8]>,
+) -> bincode::Result<()> {
+    ctx.ignore_mapping = true;
+    (deserialize)(ctx, cursor)?;
+    ctx.ignore_mapping = false;
```

Comment:
If you know the size of serialized component, you can just advance the cursor. `ignore_mapping` useful only in the default consuming function. I probably should add a doc comment about how it can be useful.